cmd/mt-index-cat/out/tpl_pattern_custom.go

```diff
@@ -0,0 +1,132 @@
+package out
+
+import (
+	"fmt"
+	"log"
+	"math/rand"
+	"os"
+	"strings"
+)
+
+type bucket struct {
+	chance int
+	fn     func(in string) string
+}
+
+// percentage chance, and function
+func patternCustom(in ...interface{}) string {
+	usage := func() {
+		fmt.Println("usage of patternCustom:")
+		fmt.Println("input | patternCustom <chance> <operation>")
+		fmt.Println("the chances need to add up to 100")
+		fmt.Println("operation is one of:")
+		fmt.Println("       pass         (passthrough)")
+		fmt.Println("       <number>rcnw (replace <number> random consecutive nodes with wildcards")
```

Comment:
consecutive nodes starting from where?

Reply:
It also needs to be made clear that <number> is a single digit

Reply:
thanks, i clarified this. if there's not enough, mt-index-cat will error out. that behavior doesn't need to be explicitly mentioned in the usage though.